soc/arm/xenvm/plat_core.S

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ *@file
+ *@brief plat/core specific init
+*/
+
+#include <toolchain.h>
+#include <linker/sections.h>
+#include <arch/cpu.h>
+
+_ASM_FILE_PROLOGUE
```

Comment:
This file is placeholder?

Reply:
Yes, actually it can be removed from the PR